### ISSUE TITLE
cpu cores estimate based on cgroups

### ISSUE THREAD

**Issue by haraldschilly** [dask/distributed#3053]
I was reading the announcement for 2.4.0 and got interested in https://github.com/dask/distributed/pull/3039 by @jcrist 

That did lead me to this part of the code:
https://github.com/dask/distributed/blob/7d017c467590c758fa4b8cb2b1193205fe5aa7ad/distributed/system.py#L62

Just by looking at it (and although I'm not an expert I think I know what's going on), I have to observations -- half way between a question and a bugreport.

1. in my docker environment I have here (ubuntu 18.04), the filename is different:

```
~$ cat /sys/fs/cgroup/cpu,cpuacct/cpu.cfs_period_us 
100000
~$ cat /sys/fs/cgroup/cpu,cpuacct/cpu.cfs_quota_us 
220000
```
in the code is that path:
```
~$ ls /sys/fs/cgroup/cpuacct,cpu/cpu.cfs_quota_us
ls: cannot access '/sys/fs/cgroup/cpuacct,cpu/cpu.cfs_quota_us': No such file or directory
```

2. The actual calculation is `int(quota / period)`. I think this should round up, not down. The point is, if you have a fraction like "2.5 cores", it will report 2 cores and keep half a core unused. It would be better if it reports 3 cores and then cgroups limits to the actual 2.5.

**Comment by jcrist** [comment]
Thanks for filing the issue. Upon further looking, it looks like OSs are inconsistent about the file path here, both `cpu,cpuacct` and `cpuacct,cpu` are possibilities - we should check for both locations. In regards to your second point, I agree we should probably use `ceil` here.

If you feel like submitting a PR for this, it would be most welcome. Otherwise I'll get to it next week.

**Comment by jcrist** [comment]
Should be fixed by #3056.
